src/ActionList/Item.tsx

```diff
@@ -295,6 +295,7 @@ const BaseVisualContainer = styled.div<{variant?: ItemProps['variant']; disabled
   height: 20px;
   width: ${get('space.3')};
   margin-right: ${get('space.2')};
+  flex-shrink: 0;
```

Comment:
Instead of this change, I think we could move the `flex-shrink: 0` to an `sx` prop on the `<BaseVisualContainer />` used in `<Item />` for single-select action lists, but this seemed like a more correct fix.

Happy to switch to the more targeted approach if we feel like it is safer.